app/src/main/java/io/github/droidkaigi/confsched2018/presentation/FragmentStateNullablePagerAdapter.java

```diff
@@ -0,0 +1,23 @@
+package io.github.droidkaigi.confsched2018.presentation;
+
+import android.support.annotation.Nullable;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentStatePagerAdapter;
+import android.view.ViewGroup;
+
+/**
+ * memo:
+ * FragmentStateNullablePagerAdapter needs to be written in Kotlin.
+ * cuz, setPrimaryItem signature is #setPrimaryItem(@NonNull ViewGroup, int, @NonNull Object),
+ * but Object come in null, so kotlin occurs NPE crash
```

Comment:
OMG 😰

Reply:
i have no idea more elegant way(java has `NonNull` annotation but actually Nullable pattern) 😢😢😢

Reply:
It's ok 🙆